en_ewt-ud-dev.conllu

```diff
@@ -11418,7 +11418,7 @@
 3	Shirley	Shirley	PROPN	NNP	Number=Sing	2	obj	2:obj	_
 4	Crenshaw	Crenshaw	PROPN	NNP	Number=Sing	3	flat	3:flat	_
 5	(	(	PUNCT	-LRB-	_	6	punct	6:punct	SpaceAfter=No
-6	X	x3-5290	NUM	GW	NumType=Card|Typo=Yes	3	parataxis	3:parataxis	_
+6	X	x3-5290	NUM	GW	NumForm=Word|NumType=Card|Typo=Yes	3	parataxis	3:parataxis	_
```

Comment:
I take it "X" stands for "extension". I see it's treated here as `goeswith` so I'm not sure what the `NumForm` should be. Arguably "X" should be a separate token joined to the numeric part with `flat`.

Reply:
Remove this NumForm as well?  Not having a NumForm at all makes it easy to search for ones we need to change later on

Reply:
Sure